components/core/src/clp_s/search/Output.cpp

```diff
@@ -65,6 +65,7 @@ bool Output::filter() {
     populate_string_queries(top_level_expr);
 
     std::string message;
+    auto const& archive_id = m_archive_reader->get_archive_id();
```

Comment:
```suggestion
    auto const archive_id = m_archive_reader->get_archive_id();
```

It's a bit weird to use a reference for an rvalue.